AutomatedTesting/Gem/PythonTests/Physics/tests/EntityComponentTests/PhysX_Collider_Component_CRUD.py

```diff
@@ -0,0 +1,127 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+def PhysX_Collider_Component_CRUD():
+    """
+    Summary:
+    Creating a PhysX Collider and setting values to all properties that are supported via the Editor Context Bus.
+
+    Expected Behavior:
+    Validates component Created, Read, Update, Delete (CRUD) and performs expected operations.
+
+    PhysX Collider Components:
+     - PhysX Collider
+
+    Test Steps:
+     1) Add an Entity to manipulate
+     2) Add a PhysX Collider Component to Entity
+     3) Set Box Shape and Child Properties
+     4) Set Capsule Shape and Child Properties
+     5) Set Cylinder Shape and Child Properties
+     6) Set Sphere Shape and Child Properties
+     7) Set General Properties
+     8) Set PhyicsAsset Shape and Child Properties
+     9) Delete Component
```

Comment:
suggestion: I know lots of the Pythons tests do work like this but it does seem to me that there's a lot going on in this one test (and therefore many reasons to fail). To debug an issue with say the Sphere Shape, you have to run through all the others right now correct? I do wonder (likely as a follow-up PR, we don't have to change this now) if we could split each step (other than 1) and 9)) to their own tests that would make finding failures easier and faster.

Reply:
We could experiment with this using the batched tests.

Reply:
That's a possibility if we can avoid the overhead of startup/shutdown for each run. It's okay to leave as-is for now, but if debugging and running these tests in future makes it difficult to diagnose failure we should consider splitting them up